### DISCUSSION TITLE
How to enable Laravel Scheduler, Queue, and Web at same time

### DISCUSSION THREAD

**Discussion by AdlyPuspakom** [php/frankenphp#903] [Q&A]
Hi there, I'm exploring FrankenPHP with Laravel and I'm impressed by its modular approach. However, I'm struggling to configure it to have both Scheduler, Queue, and Web features active concurrently. Could someone share their approach or point me to resources that explain how to set this up? I've reviewed the documentation but would appreciate any practical examples or tips from those who have successfully implemented this setup. Thanks in advance for your assistance!

**Comment by AlliBalliBaba** [comment]
In a real live application you would probably want to use supervisor to start the worker and scheduler processes. I think this repo does a really good job at showcasing how a laravel octane docker image would look like with Frankenphp/Swoole/Roadrunner: 
[https://github.com/exaco/laravel-octane-dockerfile](https://github.com/exaco/laravel-octane-dockerfile)

**Comment by fouteox** [comment]
Hi, you can see here an implementation for schedule with non root user:

https://github.com/fouteox/pingcrm-react-inertia-laravel

**Comment by AdlyPuspakom** [comment]
Thank you for sharing. I will try both of solution.
